src/content/guides/ecommerce.mdx

```diff
@@ -0,0 +1,29 @@
+---
+title: eCommerce
+description: How to process payments using Centrapay's eCommerce Platform.
+nav:
+  path: Reference/Merchant Integrations
+  order: 2
+---
+
+Centrapay enables businesses to process payments for connected Centrapay assets online. In order to process payments businesses must be able to integrate with one of our eCommerce payment flows.
+
+## Onboarding
+
+In order to request eCommerce payments, integrators will need a [merchant](https://docs.centrapay.com/api/managed-integrations/#merchant-model), a [merchant configuration](https://docs.centrapay.com/api/merchant-configs/), and a merchant-terminal [API key](https://docs.centrapay.com/api/api-keys/).
+Centrapay will provision these on behalf of the integrators. Initially the merchant and merchant configuration you receive will be configured to accept test resources.
+After completing testing and getting certified to process payments on Centrapay's eCommerce platform, Centrapay will provide you with a separate live merchant and merchant configuration which will be used to process live payments.
```

Comment:
Are you able to link the certification checklist on "getting certified"?

Reply:
I don't think we have a public list they can look at, we provide that for them when they contact us. We do have a core requirements section that we should add but I'm not 100% sure on the core requirements for eCommerce yet.

Reply:
Yeah I think the Core Requirements would be pretty similar to the [POS requirements](https://docs.centrapay.com/guides/point-of-sale/)

Difference being an ecommerce payment flow instead of QR/barcode

Reply:
Ok, we can add those at a later date when the requirements are fully defined.